src/components/IdentifierCardEditing.vue

```diff
@@ -113,20 +116,63 @@ export default defineComponent({
             default: 0
         }
     },
+    components: {
+        SchemaGuideLink
+    },
     setup (props) {
+        const linkInfo: linkInfoType = {
```

Comment:
```suggestion
        const linkInfo = {
```
rely on type inference

Reply:
resolved